api/app.ts

```diff
@@ -30,6 +30,9 @@ app.post('/changePassword', changePasswordEndpoint);
 import accountCreate from './accounts/create';
 app.post('/accounts/create', accountCreate);
 
+import changeUsername from './accounts/changeUsername';
+app.post('/accounts/changeUsername', changeUsername);
```

Comment:
I think this might need to be a PUT call since we arn't creating new rows, just updating.

Reply:
We'll leave it as a post for now. If we wish to change the endpoint, we can at a future time.

Reply:
This is not a PUT call; it does not create new data and is not idempotent.